.github/workflows/clp-core-build.yaml

```diff
@@ -12,6 +12,13 @@ on:
       - 'components/core/**'
       - '.github/workflows/clp-core-build.yaml'
   workflow_dispatch:
+  workflow_run:
+    workflows: ["Build and Publish Image with Dependencies to Build CLP Core"]
```

Comment:
To make this less error-prone, let's change this to the workflow file path or rename the workflow with a shorter name.